vite.config.ts

```diff
@@ -11,19 +11,27 @@ import { buildScheme, validateConfig } from './scripts/build-config-scheme.mjs';
 
 import postcssConfig from './postcss.config';
 import { proxyConfig } from './vite.proxy';
-import packageJson from './package.json';
 
 const relative = (folder: string) => path.resolve(__dirname, folder);
+const parseEnv = (
+  env: Record<string, string>,
+): Record<string, string> =>
+  Object.entries(env).reduce((acc, [k, v]) => {
+    try {
+      acc[k] = JSON.parse(v);
+    } catch (e) {}
```

Comment:
why to have it if nothing handled?

Reply:
Because if it just string - i want to leave it as is

Reply:
I ment to add console.log in catch block @Akiyamka